src/assets/scripts/client/info/AirportInfoView.js

```diff
@@ -0,0 +1,214 @@
+import $ from 'jquery';
+import EventBus from '../lib/EventBus';
+import SimClockController from './SimClockController';
+import AirportInfoController from './AirportInfoController';
+import { EVENT } from '../constants/eventNames';
+import { AIRPORT_INFO_TEMPLATE } from './airportInfoTemplate';
+
+/**
+ * @property STRIP_VIEW_SELECTORS
+ * @type {object<string, string}
+ * @final
+ */
+const INFO_VIEW_SELECTORS = {
+    CLOCK_LABEL: '.js-airportInfo-clock-label',
+    CLOCK_VALUE: '.js-airportInfo-clock-value',
+    WIND_LABEL: '.js-airportInfo-wind-label',
+    WIND_VALUE: '.js-airportInfo-wind-value',
+    ALTIMETER_LABEL: '.js-airportInfo-altimeter-label',
+    ALTIMETER_VALUE: '.js-airportInfo-altimeter-value',
+    ELEVATION_LABEL: '.js-airportInfo-elevation-label',
+    ELEVATION_VALUE: '.js-airportInfo-elevation-value'
+};
+
+/**
+ * Displays an information corner that displays the current time, wind speed,
+ * wind angle, altimeter, and airport elevation.
+ *
+ * @class AirportInfoView
+ */
+export default class AirportInfoView {
+    /**
+     * @for AirportInfoView
+     * @constructor
+     * @param {jQuery|HTML element}
+     */
+    constructor($element) {
+        /**
+         * Root DOM element
+         *
+         * @for AirportInfoView
+         * @property $element
+         * @type {jQuery|HTML element}
+         */
+        this.$element = $element;
+
+        /**
+         * Information div
+         *
+         * @for AirportInfoView
+         * @property $template
+         * @type {jQuery|HTML element}
+         */
+        this.$template = null;
+
+        /**
+         * @for AirportInfoView
+         * @property simClockController
+         */
+        this.simClockController = null;
+
+        /**
+         * @for AirportInfoView
+         * @property simAirportInfoController
+         * @type {AirportInfoController}
+         */
+        this.airportInfoController = null;
+
+        /**
+         * Local instance of the event bus
+         *
+         * @for AirportInfoView
+         * @property _eventBus
+         * @type {EventBus}
+         */
+        this._eventBus = EventBus;
+
+        return this.init()
+                ._createChildren()
+                ._setupHandlers()
+                ._enable();
+    }
+
+    /**
+     * @for AirportInfoView
+     * @method init
+     * @chainable
+     */
+    init() {
+        this.$template = $(AIRPORT_INFO_TEMPLATE);
```

Comment:
We need to do this without using jQuery. I think it would be `this.$element.find(AIRPORT_INFO_TEMPLATE)`?

Reply:
I am confused. StripViewModel does exactly the same:
`this.$element = $(STRIP_VIEW_TEMPLATE);`

Reply:
@erikquinn I'm all for moving away from jQuery, but we lean on it heavily for our views.  There are several things we leverage with jQuery that actually make sense for us to use it here.